hsengine/game.h

```diff
@@ -485,10 +485,10 @@ public slots:
      * board control value function.
      */
     struct BoardControlScore {
-        static const float TOUGH_MINIONS_MULT = 2.5; //!< Multiplier for the "tough minions" factor
-        static const float HAND_MULT = 1;            //!< Multiplier for the "hand" factor
-        static const float MINIONS_MULT = 1.75;      //!< Multiplier for the "minions number" factor
-        static const float SITUATION_MULT = 3.75;    //!< Multiplier for the "situation" factor
+        static constexpr const float TOUGH_MINIONS_MULT = 2.5; //!< Multiplier for the "tough minions" factor
```

Comment:
btw. Thank you for your great job and the code with excellent documentation.
Such documentation is really uncommon at github, and it makes the code much more valuable.